Docker `make build` fails

Theres a few of errors starting at:

```
docker run --rm --net=host -e USER_ID=1000 \
                -v /home/benc/go/src/github.com/fireeye/gocrack/docker/dist/hashcat:/out \
                gocrack/hashcat_shared
/bin/sh: 1: git: not found
```


Fixed by adding git to the dockerfile `apt-get install`. 

Then:

```
docker run --rm --net=host -e USER_ID=1000 \
                -v /home/benc/go/src/github.com/fireeye/gocrack/docker/dist/hashcat:/out \
                gocrack/hashcat_shared
install -m 755 -d                                    /out/share/doc/hashcat
install -m 755 -d                                    /out/share/hashcat
gcc -c -pipe -std=gnu99 -Iinclude/ -Iinclude/lzma_sdk/ -IOpenCL/ -O2 -Ideps/OpenCL-Headers/ -DWITH_HWMON src/affinity.c -o obj/affinity.NATIVE.SHARED.o -fpic
gcc -c -pipe -std=gnu99 -Iinclude/ -Iinclude/lzma_sdk/ -IOpenCL/ -O2 -Ideps/OpenCL-Headers/ -DWITH_HWMON src/autotune.c -o obj/autotune.NATIVE.SHARED.o -fpic
gcc -c -pipe -std=gnu99 -Iinclude/ -Iinclude/lzma_sdk/ -IOpenCL/ -O2 -Ideps/OpenCL-Headers/ -DWITH_HWMON src/benchmark.c -o obj/benchmark.NATIVE.SHARED.o -fpic
install: cannot create directory '/out/share': Permission denied
gcc -c -pipe -std=gnu99 -Iinclude/ -Iinclude/lzma_sdk/ -IOpenCL/ -O2 -Ideps/OpenCL-Headers/ -DWITH_HWMON src/bitmap.c -o obj/bitmap.NATIVE.SHARED.o -fpic
src/Makefile:328: recipe for target 'install_docs' failed
make: *** [install_docs] Error 1
make: *** Waiting for unfinished jobs....
install: cannot create directory '/out/share': Permission denied
make: *** [install_shared] Error 1
src/Makefile:358: recipe for target 'install_shared' failed
```

Permissions on dist appear to be created with root ? Chowned that back to my user (1000). 

I also get some additional errors but I think that is due to UMASK           027

--- a/docker/Dockerfile.hashcat.ubuntu
+++ b/docker/Dockerfile.hashcat.ubuntu
@@ -7,6 +7,7 @@ LABEL AUTHOR=${AUTHOR}
 RUN apt-get update && apt-get install -y --no-install-recommends \
         make \
         build-essential \
+        git \
         patch && \
     rm -rf /var/lib/apt/lists/*
 

